ocpmodels/trainers/forces_trainer.py

```diff
@@ -317,20 +318,27 @@ def train(self):
                     split="val", epoch=epoch,
                 )
 
-            if not self.is_debug:
-                save_checkpoint(
-                    {
-                        "epoch": epoch + 1,
-                        "state_dict": self.model.state_dict(),
-                        "optimizer": self.optimizer.state_dict(),
-                        "normalizers": {
-                            key: value.state_dict()
-                            for key, value in self.normalizers.items()
+            if (
+                val_metrics.meters["force_z/mae"].global_avg
```

Comment:
Now is probably a good time to add "force_avg/mae" and use that here as well?

Reply:
Yeah, working on it, not as part of this but as a separate `Evaluator` class.. I'd suggest going off of this for your experiments for now (ok to not merge this immediately) and I'll update it later.

Reply:
Will go ahead and merge for now, and the `Evaluator` class can be its own PR. Will be easier to keep in sync with all these PRs.